governance/handler.go

```diff
@@ -168,12 +168,28 @@ func (gov *Governance) checkKey(k string) bool {
 	return false
 }
 
+func (gov *Governance) checkValue(key string, val interface{}) bool {
```

Comment:
@mckim19 I think `check.Validator` does the same job. Could you move below logic to `check.Validator`?

Reply:
thanks for your advice. I will move the logic to ckeck.Validator.